advanced_functionality/tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb

```diff
@@ -22,6 +22,13 @@
     "__Note__: Compare this with the [tensorflow bring your own model example](../tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb)"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    EPOCHS = 50
</pre><p>What is the approximate runtime with 50 epochs? If it's more than ~1 hour, please lower the number of epochs and add a note that the customer can increase for better accuracy.</p><p> </p><p>I see you're also removing lines of explanatory text. As you revise the code, add new explanations describing what this revised version does. The examples should educate the customer as well as providing them with working code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
It trains in about 5 seconds - the benefits of using a super-simple model definition. I've added a note to the markdown.